api/scripts/_template.js

```diff
@@ -0,0 +1,38 @@
+require('dotenv').config();
+const { performance } = require('perf_hooks');
+const logger = require('../lib/infrastructure/logger');
+const { knex, disconnect } = require('../db/knex-database-connection');
+
+const doSomething = async ({ throwError }) => {
+  if (throwError) {
+    throw new Error('An error occurred');
+  }
+  const data = await knex.select('id').from('users').first();
+  return data;
```

Comment:
suggestion: Pour alleger le code
```suggestion
  return knex.select('id').from('users').first();
```

Reply:
Je trouve que retourner une promesse n'est pas facile à comprendre, je préfère laisser tel quel.